app/steps/ui/executors/number/index.js

```diff
@@ -14,6 +15,7 @@ module.exports = class ExecutorsNumber extends ValidationStep {
     }
 
     createExecutorList(ctx, formdata) {
+        const executorsWrapper = new ExecutorsWrapper(formdata.executors);
         ctx.list = get(ctx, 'list', []);
```

Comment:
It's not necessarily in the scope of this but now we're using the `ExecutorsWrapper` we have a method to get the executor list, for example:

```
ctx.list = executorsWrapper.executors();
```

This also means we don't need to require `lodash`.